02-household-queries/qt_summaries-gpt-3.5-turbo-instruct-k_4.json

```diff
@@ -0,0 +1,851 @@
+                "summary": "In cases where all adults in the SNAP household are ineligible non-citizens but have eligible children, the adult must still be listed as the head of household and their income/resources will be counted for determining eligibility."
+            }
+        }
+    },
+    {
+        "id": 3,
+        "question": "The client rents a room in a house. He said he wants to apply on his own, but then he admitted he shares a kitchen with his roommates and they often cook and eat together. Do the roommates need to be included?",
+        "guru_cards": [
+            "What if my client shares meals but wants to apply for food stamps on their own?"
+        ],
+        "recall": 1.0,
+        "extra_cards": 5,
+        "derived_questions": [
+            "Does the client share a kitchen with his roommates?",
+            "Do the roommates often cook and eat together in the shared kitchen?"
+        ],
+        "all_retrieved_cards": {
+            "What if a client does not want to provide information about other household members?": {
+                "score_sum": 0.5376032753254323,
+                "derived_questions": [
+                    "Does the client share a kitchen with his roommates?"
+                ],
+                "entire_card": "What if a client does not want to provide information about other household members?\nWe always want to try and collect as much information about everyone living in the household, but sometimes clients are not comfortable disclosing information about people living there who are non-applicants.\nIf a client is living with non-mandatory household members, it is helpful to include basic information. When other household members are not included, it can pose delays down the line when caseworkers uncover other household members. When we have an idea of the composition of the household, it allows for faster troubleshooting.\nFOR FOOD STAMP BENEFITS\nIn scenarios where there are people who are\nNOT in the client's SNAP household\n(roommates/boarders/family members who share less than 50% of the grocery budget) ideally, you will want to collect full name, birth date, gender and marital status for everyone not applying.  You will\nat least\nneed to collect names and approximate ages of other people in the household.\nWe can proceed with the application if the client is insistent about not including them. We should still leave a portfolio note explaining that other household members reside with the client, but the client requested they not be included.\nFor example: Client has 3 additional roommates, but requested they not be listed on the application. Client does not share a food budget with the other members and they are not mandatory household members.",
+                "summary": "It is important to collect information about all household members, even if they are not mandatory, as it can help with troubleshooting and avoiding delays in the future."
```

Comment:
**Relevance to the question:** 🟢 Directly relevant

**Correctness as a summary of the Guru card**: 🟡 Missing Info
- The summary is correct but the Guru card provides specific guidance about the minimal fields to collect, as well as guidance about how to proceed if the client insists on not including additional household members

**Helpfulness**: 🟡 Neutral
- The summary is partially helpful in that it reinforces the importance of collecting the roommates' information, but it does not clarify whether the roommates should be considered mandatory to include on the application